### PR TITLE
Fix #40966

### LINKED ISSUE #40966: buttons in 'Shared with You' view always use default code path
We missed a couple of places where users of [our OCM app](https://github.com/pondersource/oc-opencloudmesh) needs to be able to divert from the default code path using config settings, sorry! They are the 'accept' and the 'decline' button in the 'Shared with You' view.

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

**Comment by phil-davis** [comment]
@michielbdejong somehow the CLA assistaqnt does not think that you have clicked the buttons to "sign" the CLA. I am surprised about that! But can you please click the button...

**Comment by pako81** [comment]
@michielbdejong please also create a changelog for this.

**Comment by DeepDiver1975** [comment]
> @michielbdejong somehow the CLA assistaqnt does not think that you have clicked the buttons to "sign" the CLA. I am surprised about that! But can you please click the button...

CLA of Yashar is missing .... he is the author ...
image

File: apps/files_sharing/lib/Controller/RemoteOcsController.php [hunk 1]
```diff
@@ -111,6 +111,25 @@ public function acceptShare($id) {
 			);
 		}
 
+		// Allow the Federated Groups app to overwrite the behaviour of this endpoint
+		$managerClass = $this->config->getSystemValue('sharing.groupExternalManager');
```
Comment (jvillafanez) [inline apps/files_sharing/lib/Controller/RemoteOcsController.php:115]: There is some duplication getting the external manager. We can move that piece of code in a private method in order to reduce the duplication. I think this also affects to some code in this class not present in this PR.

**Pull request closed without merging**